### PR TITLE
chore: add Trust Onye

### PR DESCRIPTION
**Pull request by code-with-onye**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1566"></a>

### CODE REVIEW

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
Hey there! Please revert the changes in the `package-lock.json` file.
Let us know if you need any help with that! 🙂


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Tags:** Not maintainable

**Comment by code-with-onye** [comment]
Yes, please I need help

File: src/Components/Milestones.js [hunk 1]
```diff
@@ -26,7 +26,8 @@ function Milestones({ milestones }) {
     <Card
       title={milestone.title}
       subTitle={milestone.date}
-      className="my-5 md:mx-5 shadow-8"
+      className="my-5 md:mx-5 shadow-8  "
+      style={{ borderRadius: '20px' }}
```
Comment (kumarsonsoff3) [inline src/Components/Milestones.js:30]: Another PR to add the border radius is already underReview! So Revert the changes from `Milestone.js`
```suggestion
```

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Not reusable

File: src/Components/Milestones.js [hunk 2]
```diff
@@ -26,7 +26,8 @@ function Milestones({ milestones }) {
     <Card
       title={milestone.title}
       subTitle={milestone.date}
-      className="my-5 md:mx-5 shadow-8"
+      className="my-5 md:mx-5 shadow-8  "
```
Comment (kumarsonsoff3) [inline src/Components/Milestones.js:29]:
```suggestion
      className="my-5 md:mx-5 shadow-8"
```

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]
Please make the following changes **++** Rename your file name to your `username.json` which should be `code-with-onye.json`.

I've specified the changes needed, Let us know if you need any other help with these! 

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Not maintainable

**Review by Pradumnasaraf — CHANGES_REQUESTED** [review]
Hey, please revert the `package-lock.json` changes. If you need help, do let's know

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Not reusable

**Review by kumarsonsoff3 — APPROVED** [review]
Hopefully good to go now! 🚀

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Elegant, Maintainable

**Review by EmmaDawsonDev — APPROVED** [review]

**Pull request merged**